pkg/action/mount.go

```diff
@@ -420,6 +421,13 @@ func SelinuxRelabel(cfg *types.RunConfig, spec *types.MountSpec) error {
 
 	paths := []string{}
 	paths = append(paths, spec.Ephemeral.Paths...)
+	for _, vol := range append(spec.Volumes, &spec.Persistent.Volume) {
```

Comment:
This is to also relabel `/oem` and other eventual volumes mounted as RW.

Reply:
Nice, I was thinking of this, but would it not impact the persistent volume as well?

Reply:
Well, the idea is to also relabel the persistent volume. The only case in which is not relabeled is when this is mounted under `/run`. I have not fully tested it yet, but I'd like to be capable to set the persistent volume mountpoint to `/var`, in that case we should also relabel `/var`.

To be honest to it looks like we might need to revisit this part as long as we gain experience and encounter new corner cases. It is hard to generalize.